cloudflare/src/endpoints/argo_tunnel/mod.rs

```diff
@@ -0,0 +1,29 @@
+use chrono::{offset::Utc, DateTime};
+use uuid::Uuid;
+
+use crate::framework::response::ApiResult;
+
+pub mod create_tunnel;
+
+/// A Named Argo Tunnel
+/// This is an Argo Tunnel that has been created. It can be used for routing and subsequent running.
+/// https://api.cloudflare.com/#argo-tunnel-properties
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
+pub struct Tunnel {
```

Comment:
Suggestion/nit: Move these into a `data_structures.rs` file in this module, and then have this mod.rs `pub use datastructures::*` so all the datatypes are in the main `argo_tunnel` namespace.